services/horizon/internal/ingest/verify.go

```diff
@@ -193,6 +193,9 @@ func (s *system) verifyState(verifyAgainstLatestCheckpoint bool) error {
 			case xdr.LedgerEntryTypeLiquidityPool:
 				lPools = append(lPools, key.LiquidityPool.LiquidityPoolId)
 				totalByType["liquidity_pools"]++
+			case xdr.LedgerEntryTypeContractData, xdr.LedgerEntryTypeConfigSetting:
+				// TODO: add support for protocol 20 ledger entries
```

Comment:
```suggestion
				// TODO: #4617 , add support for protocol 20 ledger entries
```

I'm working on #4617 for addtl test coverage, can pick it up from there.